app/grandchallenge/evaluation/models.py

```diff
@@ -475,6 +483,25 @@ class Phase(FieldChangeMixin, HangingProtocolMixin, UUIDModel):
         blank=True,
         help_text="The output interfaces that the algorithms for this phase must use",
     )
+    selectable_gpu_type_choices_evaluation: list[str] = models.JSONField(
+        blank=True,
```

Comment:
I'm not sure if we want to allow `blank`, does `[]` count as `blank`? If so, then we want to keep it, if not, we want to deny it.

Reply:
`[]` counts as blank. But I'm denying it, because we only have one list now; otherwise there are no options.

Reply:
What about CPU only: `[""]`?